### PR TITLE
Customizable web page login language

### PR DESCRIPTION
**Pull request by onevcat**
@wangzun123 This PR adds support for setting language of the web page used for web login. Please check it when you have time.

### CODE REVIEW

File: LineSDK/LineSDK/Login/LoginManager.swift [hunk 1]
```diff
@@ -342,3 +354,54 @@ extension LoginManager {
...
+        public static let portugueseEuropean = WebPageLanguage(rawValue: "pt-PT")
+        /// The Russian langauge.
+        public static let russian = WebPageLanguage(rawValue: "ru")
+        /// The Thai langauge.
+        public static let thai = WebPageLanguage(rawValue: "th")
+        /// The Turkish langauge.
+        public static let turkish = WebPageLanguage(rawValue: "tr")
+        /// The Vietnamese langauge.
+        public static let vietnamese = WebPageLanguage(rawValue: "vi")
+        /// The SimplifiedC Chinese langauge.
```
Comment (eJamesLin) [inline LineSDK/LineSDK/Login/LoginManager.swift]: little typo of SimplifiedC / TraditionalC~

Comment (onevcat) [reply LineSDK/LineSDK/Login/LoginManager.swift]: Oops

Comment (onevcat) [inline LineSDK/LineSDK/Login/LoginManager.swift]:
```suggestion
        /// The Simplified Chinese langauge.
```

File: LineSDK/LineSDK/Login/LoginManager.swift [hunk 2]
```diff
@@ -342,3 +354,54 @@ extension LoginManager {
...
+        public static let chineseSimplified = WebPageLanguage(rawValue: "zh-Hans")
+        /// The TraditionalC  Chinese langauge.
```
Comment (onevcat) [inline LineSDK/LineSDK/Login/LoginManager.swift]:
```suggestion
        /// The Traditional  Chinese langauge.
```

**Comment by onevcat** [comment]
@eJamesLin All fixed. It should be ready now!

**Pull request merged**